plugins/generator-1.19.4/forge-1.19.4/utils/triggers.java.ftl

```diff
@@ -217,6 +217,19 @@
 </#if>
 </#macro>
 
+<#macro piglinNeutral procedure="">

```

Comment:
Indentation should be improved (also for 1.20.1)

Reply:
This does not look right to me either:

![image](https://github.com/MCreator/MCreator/assets/16374228/eadf29e3-9592-4c17-a393-f7c4f982db7b)